woocommerce/woocommerce
# Disable tabs in parent product page with variations

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





When a variation is added to a product, a message will be displayed upon visiting the Pricing, Inventory, or Shipping tabs.

The message reads:
`This product has options, such as size or color. You can now manage each variation's price and other details individually. [Go to Variations]`

**Design**

![Screenshot 2023-08-24 at 09 38 08](https://github.com/woocommerce/woocommerce/assets/1314156/41cb0e09-de54-4391-89ef-1763b5e0e6c4)

AkVNGImLgSqCObTQ3idVn7-fi-2780_278412

**Epic**

#36286

**ACs**

- [x] When a product has at least one variation, show a message at the top of the form inside the Pricing, Inventory, and Shipping tabs.
- [x] The message reads: `This product has options, such as size or color. You can now manage each variation's price and other details individually. [Go to Variations]`
- [x] The `Go to Variations` button takes the user to the `Variations` tab.
- [x] The message is not dismissible.

Closes #39459.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the product blocks editor under WooCommerce -> Settings -> Advanced -> Features.
2. Make sure feature `product-variation-management` is enabled under `Features` tab from `/wp-admin/tools.php?page=woocommerce-admin-test-helper`.
3. Navigate to the add product page.
4. Go to `Variations` and add a variation.
5. Visit the `Pricing` tab. The following message will be visible:
```
This product has options, such as size or color. You can now manage each variation's price and other details individually.
```
6. Verify that the `Go to Variations` button works as expected.
7. Go to the `Inventory` tab and verify that the mentioned text in point 4 is visible. Also, test the `Go to Variations` button.
8. Go to the `Shipping` tab and verify that the mentioned text in point 4 is visible. Also, test the `Go to Variations` button.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
